### ISSUE TITLE
Color data

### ISSUE THREAD

**Issue by noraj** [lumean/svg-graph2#20]
I searched in the code and saw no way to choose color without providing a CSS stylesheet.

**Comment by lumean** [comment]
yes, this is how it currently works. Please feel free to submit a PR or add your suggestions here if you have improvement ideas. My time for this project is currently quite limited, so please don't expect too much from my side. I will mark this as a feature request
